whois-nrtm4/src/main/java/net/ripe/db/nrtm4/util/NrtmFileUtil.java

```diff
@@ -0,0 +1,49 @@
+package net.ripe.db.nrtm4.util;
+
+import com.google.common.hash.Hashing;
+import net.ripe.db.nrtm4.publish.PublishableDeltaFile;
+import net.ripe.db.nrtm4.publish.PublishableSnapshotFile;
+import org.springframework.stereotype.Service;
+
+import java.nio.charset.StandardCharsets;
+import java.util.Random;
+import java.util.UUID;
+
+import static net.ripe.db.nrtm4.NrtmConstants.DELTA_PREFIX;
+import static net.ripe.db.nrtm4.NrtmConstants.SNAPSHOT_PREFIX;
+
+
+@Service
+public class NrtmFileUtil {
+
+    private static final String CHARS = "0123456789abcdef";
+    private static final int RAND_STRING_LENGTH = 40;
+
+    public String fileName(final PublishableDeltaFile file) {
+        return String.format("%s.%d.%s.json", DELTA_PREFIX, file.getVersion(), randomHexString());
+    }
+
+    public String fileName(final PublishableSnapshotFile file) {
+        return String.format("%s.%d.%s.json", SNAPSHOT_PREFIX, file.getVersion(), randomHexString());
+    }
+
+    public String hashString(final String payload) {
+        return Hashing.sha256()
+            .hashString(payload, StandardCharsets.UTF_8)
```

Comment:
We already use DigestUtils class to generate md5Hex , can we use the same library to generate sha256?

Reply:
Or use built-in `MessageDigest.getInstance("SHA-256");`

Reply:
done